app/assets/javascripts/backbone/views/timeline_select_steps_view.coffee

```diff
@@ -9,44 +9,40 @@ class MS.Views.TimelineSelectStepsView extends Backbone.View
     'click #uncheck-all-ctrl': 'uncheckAll'
     'click #check-completed-ctrl': 'checkCompleted'
     'click #check-incomplete-ctrl': 'checkIncomplete'
- 
-  checkAll: (e) ->
-    $control = $(e.currentTarget)
-    $inputs = $control.closest('.timeline').find('.select-step')
-    @.checkItems($inputs)
 
-    $masterCheckbox = $('#choose-all')
-    @.checkItems($masterCheckbox)
+  checkAll: () ->
+    @.toggleAll(true)
 
-    @.rememberChecked(e)
+  uncheckAll: () ->
+    @.toggleAll(false)
 
-  uncheckAll: (e) ->
-    $control = $(e.currentTarget)
-    $inputs = $control.closest('.timeline').find('.select-step')
-    @.uncheckItems($inputs)
+  checkCompleted: () ->
+    $inputs = @$el.find('.select-step.completed')
+    @.toggleSubset(true, $inputs)
 
-    $masterCheckbox = $('#choose-all')
-    @.uncheckItems($masterCheckbox)
+  checkIncomplete: () ->
+    $inputs = @$el.find('.select-step:not(.completed)')
```

Comment:
Hi @hooverlunch and @emitche I had to change this based on the changes in #158.

Used to use the classes `.completed-item` and `.incompleted-item`. The change was to a `:not()` selector since the incomplete class is now gone. I thought I would check with you both first if this is a good idea? I usually like to avoid the usage of the `not` selector. We could potentially add additional states in the future requiring this to change.